app/src/main/java/org/fossasia/phimpme/utilities/Constants.java

```diff
@@ -26,5 +26,8 @@
 
     public static String IMGUR_HEADER_CLIENt = "Client-ID";
     public static String IMGUR_HEADER_USER = "Bearer";
+
+    public final static String APP_KEY = "9qgi9rcomdglx8f";
```

Comment:
Not sure if exposing this is a good idea.
Maybe we can have an internal testing with this api key and if it works well then we can remove this and replace it with a dummy string.